Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2652,12 +2706,20 @@ else if (thisReq_ instanceof ServerRequestLogout) {
                                 if (thisReq_ instanceof ServerRequestInitSession) {
                                     initState_ = SESSION_STATE.INITIALISED;
                                     // Publish success to listeners
-                                    thisReq_.onRequestSucceeded(serverResponse, branchReferral_);
```

Comment:
Can we report the `onRequestSucceeded()` before `checkForAutoDeepLinkConfig` as it was earlier